safe-room.lic

```diff
@@ -72,6 +72,9 @@ class SafeRoom
   def wait_for_healing
     Flags.reset('healthy')
     Flags.reset('moved')
+    
+    lie_down
```

Comment:
Do we need to lie down before tending our wounds?

We're already lieing down on line 80.